en/docs/reference/config-catalog.md

```diff
@@ -4443,3 +4445,1562 @@ enable_email_domain = true</code></pre>
+                            <span class="badge-required">Optional</span>
+                            <p>
+                                This includes the bandwidth configurations related to the "SMB" billing package.
+                            </p>
+                        </div>
+                        <div class="params-wrap">
+                            <div class="param">
+                                <div class="param-name">
+                                    <span class="param-name-wrap"> <code>limit</code> </span>
+                                </div>
+                                <div class="param-info">
+                                    <div>
+                                        <p>
+                                            <span class="param-type string"> string </span>
+                                        </p>
+                                    </div>
+                                    <div class="param-description">
+                                        <p>The highest bandwidth value that can be allocated for the "SMB" billing package. </p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="param">
+                                <div class="param-name">
+                                    <span class="param-name-wrap"> <code>overuse_charge</code> </span>
+                                </div>
+                                <div class="param-info">
+                                    <div>
+                                        <p>
+                                            <span class="param-type string"> string </span>
+                                        </p>
+                                        <div class="param-default">
+                                            <span class="param-default-value">Default: <code>"0"</code></span>
+                                        </div>
+                                    </div>
+                                    <div class="param-description">
+                                        <p> The charge for over usage based on the billing package. </p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</div>
+
+<div class="mb-config-catalog">
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="46" type="checkbox" id="_tab_46">
+                <label class="tab-selector" for="_tab_46"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[[multi_tenancy.billing.package.custom]]

```

Comment:
Isn't it incorrect to add double `[[ ]]` here?

<img width="794" alt="Screenshot 2020-10-21 at 20 56 39" src="https://user-images.githubusercontent.com/5195851/96751308-a5363400-13ea-11eb-8a27-deef5a5d3236.png">

Reply:
Yes, it is incorrect. I have corrected that.